introduction.md

```diff
@@ -0,0 +1 @@
+Running codeschool challenge. Working through pull request challenge.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.